voting-webapp/experiment.yaml

```diff
@@ -8,48 +8,44 @@ spec:
     value: "100"
   parameters:
   - name: worker_cpu
+    baseline: 1000
     min: 100
     max: 1000
-    baseline: 1000
   - name: voting_memory
+    baseline: 2000
     min: 200
     max: 2000
-    baseline: 2000
   - name: voting_cpu
+    baseline: 1000
     min: 100
     max: 1000
-    baseline: 1000
   - name: voting_replicas
+    baseline: 3
     min: 1
     max: 4
-    baseline: 3
   - name: db_memory
+    baseline: 2000
     min: 200
     max: 2000
-    baseline: 2000
   metrics:
   - name: throughput
     minimize: false
     type: jsonpath
     query: '{.total}'
-    path: '/metrics'
-    port: 5000
-    selector:
-      matchLabels:
-        component: result-exporter
+    url: 'http://result-exporter:5000/metrics'
```

Comment:
@jgustie I had to take out the `throughput` metric, then put it back in after the `fix` was run. Do you approve the value of URL I used? I haven't run this experiment yet, although it applies fine.

Reply:
Yes, this would have been a change that fix cannot fix because it's can't know the label selector `component=result-exporter` matches a `Service` that is also named `result-exporter`. This looks correct to me.